plugins/parsers/json/README.md

```diff
@@ -18,6 +18,10 @@ ignored unless specified in the `tag_key` or `json_string_fields` options.
   ## https://github.com/influxdata/telegraf/blob/master/docs/DATA_FORMATS_INPUT.md
   data_format = "json"
 
+  ## When strict is true and a JSON array is being parsed, all objects within the
+  ## array must be valid
+  strict = false
```

Comment:
You will need to modify the `plugins/parsers/registry.go` file to fish this option in.  Right now there is no way to set it.  We should name it `json_strict` since this available directly at the plugin scope.